### DISCUSSION TITLE
importError: no module named redis

### DISCUSSION THREAD

**Discussion by estherli97** [cyberbotics/webots#4390] [Q&A]
Hi,
I try to use python to create a controller in webots. But I found that it is impossible to import 'redis' in my script. Actually it works fine in my other python files. It just could not work in this script.Is there anything I need to change in webots python makefile or perhaps in my environment variables?

![importError no module named redis](https://user-images.githubusercontent.com/96454924/159697146-981345e7-00ff-4084-87bc-1cd1b931516b.png)

Thank you!

**Comment by omichel** [comment]
Are you sure the python version used by Webots is the same as the one you run with your other python files? The python command used by Webots can be defined in the Webots preferences.
